### PR TITLE
[IMP] hr,lunch,base: improve demo data

### PR DESCRIPTION
**Pull request by ravb-odoo**
This commit improves the quality and accuracy of the demo data.

task-3493631

### CODE REVIEW

**Comment by sofiagvaladze** [comment]
@ravb-odoo the task has conflict. Please rebase :smile:

File: odoo/addons/base/data/res_users_demo.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
             <field name="company_name">YourCompany</field>
             <field name="street">3575  Buena Vista Avenue</field>
             <field name="city">Eugene</field>
-            <field name="state_id"  model="res.country.state" search="[('code','ilike','OR')]"/>
+            <field name="state_id"  model="res.country.state" search="[('code','=like','OR')]"/>
```
Comment (sofiagvaladze) [inline odoo/addons/base/data/res_users_demo.xml:11]: could be just `=` as well, IMO

**Comment by ravb-odoo** [comment]
Hello @sofiagvaladze 
Conflict have been resolved.

Thank You :smile:

**Comment by sofiagvaladze** [comment]
robodoo r+

**Pull request closed without merging**